### ISSUE TITLE
Rehearsal marks

### ISSUE THREAD

**Issue by joeberkovitz** [w3c/mnx#112]
This issue notes the need for rehearsal marks.

MusicXML supports these via the `rehearsal` element.

**Comment by cyrilcoutelier** [comment]
I think it could be interesting to have some semantic, wether the content of a rehearsal mark is custom, if it displays the measure number, or if it is ordered alphabetically. For the purpose of having 
it correctly updated by the software.

**Comment by joeberkovitz** [comment]
@cyrilcoutelier Good point. However, as this puts a new requirement on consumer apps -- to follow a specified algorithm that determines the content of non-custom marks -- we should consider whether that's a good tradeoff. Instead, apps can "notice" that a given rehearsal mark follows a sequence that the app supports, and import it as automatic instead of custom. This seems maybe more flexible in that it doesn't restrict automatic rehearsal marks to a restricted set of algorithms. Perhaps there are problems with that approach, though.

**Comment by cyrilcoutelier** [comment]
Maybe there could be an `other` field in the algorithm enum, opening the door to other algorithms/interpretations, while allowing the standard to provide some semantic on the "common" patterns. A bit like the `other-dynamics` element. 
Having some semantic would also make it easier to mix the types, although I am not sure if this is a common practice.
